lib/active_merchant/billing/gateways/cyber_source.rb

```diff
@@ -841,49 +840,18 @@ def subsequent_nt_apple_pay_auth(source, options)
       end
 
       def add_auth_network_tokenization(xml, payment_method, options)
-        return unless network_tokenization?(payment_method)
-
-        commerce_indicator = 'internet' if subsequent_nt_apple_pay_auth(payment_method.source, options)
-
         brand = card_brand(payment_method).to_sym
-
         case brand
-        when :visa
-          xml.tag! 'ccAuthService', { 'run' => 'true' } do
-            xml.tag!('cavv', payment_method.payment_cryptogram) unless commerce_indicator
-            xml.commerceIndicator commerce_indicator.nil? ? ECI_BRAND_MAPPING[brand] : commerce_indicator
-            xml.tag!('xid', payment_method.payment_cryptogram) unless commerce_indicator
-            xml.tag!('reconciliationID', options[:reconciliation_id]) if options[:reconciliation_id]
-          end
-        when :master
+        when :visa, :master, :american_express
           xml.tag! 'ccAuthService', { 'run' => 'true' } do
-            xml.commerceIndicator commerce_indicator.nil? ? ECI_BRAND_MAPPING[brand] : commerce_indicator
-            xml.tag!('reconciliationID', options[:reconciliation_id]) if options[:reconciliation_id]
-          end
-        when :american_express
-          cryptogram = Base64.decode64(payment_method.payment_cryptogram)
-          xml.tag! 'ccAuthService', { 'run' => 'true' } do
-            xml.tag!('cavv', Base64.encode64(cryptogram[0...20]))
-            xml.tag!('commerceIndicator', ECI_BRAND_MAPPING[brand])
```

Comment:
Why remove the commerce_indicator?

Reply:
Because it's optional and I decided just to remove it.

Reply:
Hmmm my concern with removing it is that we needed to mark commerce_indicator as `internet` for recurring AP/GP recently. Do we know what the default commerce_indicator is?

Reply:
Okay, I'll add it back then.

Reply:
Where did you see it was optional? We previously had a brand_mapping for this field so I'm not sure if that mapping was needed or if hard coding `internet` is correct